sct.py

```diff
@@ -677,6 +677,29 @@ def run_test(argv, backend, config, logdir):
                   failfast=False, buffer=False, catchbreak=True)
 
 
+@cli.command('run-pytest', help="Run tests using pytest")
```

Comment:
which tests this CLI command will run?

Reply:
it will run any pytest you point to sct.py run-pytest scylla_operator will run pytests from scylla_operator